kolibri/core/device/serializers.py

```diff
@@ -266,6 +267,27 @@ class Meta:
     def create(self, validated_data):
         raise serializers.ValidationError("Device settings can only be updated")
 
+    def update(self, instance, validated_data):
+        if "extra_settings" in validated_data:
+            updated_extra_settings = validated_data.get("extra_settings")
+            initial_extra_settings = getattr(instance, "extra_settings", "{}")
+
+            if updated_extra_settings != initial_extra_settings:
+                automatic_download_enabled = updated_extra_settings.get(
+                    "enable_automatic_download"
+                )
+                if (
+                    automatic_download_enabled
+                    and automatic_download_enabled
+                    != initial_extra_settings.get("enable_automatic_download")
+                ):
+                    automatic_synchronize_content_requests_and_import.enqueue()
+
```

Comment:
Should we cancel any existing tasks when it is disabled?

Reply:
Wouldn't hurt!